File: src/main/java/com/ninebudget/component/AccountComponent.java
package com.ninebudget.component;

import com.ninebudget.model.Account;
import com.ninebudget.model.Component;
import com.ninebudget.model.ComponentException;

public class AccountComponent implements Component<Account> {
    @Override
    public Account get(Account object) throws ComponentException {
        return null;
    }

    @Override
    public void update(Account object) throws ComponentException {

    }

    @Override
    public void create(Account object) throws ComponentException {

    }

    @Override
    public void deactivate(Account object) throws ComponentException {

    }
}

File: web/src/app/component/account/account.component.ts
import { Component, OnInit } from '@angular/core';
import {Account} from '../../model/account';
import {AccountService} from '../../service/account.service';

@Component({
  selector: 'app-account',
  templateUrl: './account.component.html',
  styleUrls: ['./account.component.scss']
})
export class AccountComponent implements OnInit {
  account: Account;

  constructor(private accountService: AccountService) {}

  ngOnInit() {
    const account: Account = new Account();
    account.id = 2;

    this.accountService.get(account).subscribe(data => {
      this.account = data;
    });
  }
}
